test/unit/base/ControllerSpec.scala

```diff
@@ -43,4 +44,8 @@ trait ControllerSpec
     FakeRequest("POST", "")
       .withJsonBody(body)
       .withCSRFToken
+
+  protected def mockArrivalJourney(): Unit = withCaching(Choice.choiceId, Some(Choice(AllowedChoiceValues.Arrival)))
```

Comment:
I think it would read better in unit tests if this was called something like:
"givenIAmOnTheArrivalJourney"
"givenAUserOnTheArrivalJourney"
etc

When I read the unit tests and I see "mockArrivalJourney" my first thought at the moment is "what does that do?"

Reply:
Okay, `givenAUserOnTheArrivalJourney` sounds better for me

Reply:
Similar for the method below

Reply:
Fixed